tasks/sep/006-1-sc-changes/SignFromJson.s.sol

```diff
@@ -197,9 +196,9 @@ contract SignFromJson is OriginalSignFromJson {
         require(systemConfig.l1StandardBridge().code.length != 0, "1901");
         require(EIP1967Helper.getImplementation(systemConfig.l1StandardBridge()).code.length != 0, "1902");
 
-        require(systemConfig.l2OutputOracle() == proxies.L2OutputOracle, "2000");
-        require(systemConfig.l2OutputOracle().code.length != 0, "2001");
-        require(EIP1967Helper.getImplementation(systemConfig.l2OutputOracle()).code.length != 0, "2002");
+        // require(systemConfig.l2OutputOracle() == proxies.L2OutputOracle, "2000");
+        // require(systemConfig.l2OutputOracle().code.length != 0, "2001");
+        // require(EIP1967Helper.getImplementation(systemConfig.l2OutputOracle()).code.length != 0, "2002");
```

Comment:
Depending on the execution order on mainnet between this task and FPs, the L2OO may or may not exist. Given the scope of this proposal, I don't think checking for L2OO vs. DisputeGameFactory makes much difference, so I've commented out all L2OO checks

Reply:
I'm fine with that.